ecs-microservice/versions.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "~> 3.21.0"
```

Comment:
Nitpick: to be more flexible wrt. to upstream root modules, HashiCorp recommends using `>=` (the minimum required version) when specifying requirements inside a reusable module